helpers/userEvents.js

```diff
@@ -0,0 +1,49 @@
+var MobileDetect = require('mobile-detect');
+var UserAnalytics = require('user-analytics');
+
+function UserEvents() {}
+
+UserEvents.trackSignup = function(req, user) {
+    const endpoint = process.env.USER_ANALYTICS_ENDPOINT;
+    const token = process.env.USER_ANALYTICS_TOKEN;
+    const streamName = process.env.USER_ANALYTICS_STREAM_NAME;
+    const env = process.env.NODE_ENV;
+    console.log('options', endpoint, token);
+    const userAnalytics = new UserAnalytics({
+        token: token,
+        endpoint: endpoint,
+    });
+    const mobileDetect = new MobileDetect(req.headers['user-agent']);
+    const data = {
+        distinct_id: '',
```

Comment:
Should this possibly be `user.tracking_id`? I'm fairly certain that `user.id` would not be kosher here for GDPR reasons. Thoughts?

Can `user.tracking_id` be accessed from anywhere else? It isn't in the response of the `getAccounts` JSON-RPC method.

Reply:
a user of faucet won't have an on-chain account at this point -- if we want to use some sort of id here we can use our `X-Request-Id` (see app.js)